core/src/test/scala/org/http4s/metrics/MetricsOpsSpec.scala

```diff
@@ -1,21 +1,27 @@
-package org.http4s.metrics.prometheus
+package org.http4s.metrics
 
 import cats.effect.IO
 import cats.implicits._
 import java.util.UUID
 import org.http4s._
 import org.scalacheck.{Arbitrary, Gen}
-import Prometheus.classifierFMethodWithOptionallyExcludedPath
+import MetricsOps.classifierFMethodWithOptionallyExcludedPath
+import org.specs2.ScalaCheck
+import org.specs2.mutable.Specification
 
-object PrometheusSpec {
+object MetricsOpsSpec {
 
   private implicit val arbUUID: Arbitrary[UUID] =
     Arbitrary(Gen.uuid)
+
+  private implicit val http4sTestingArbitraryForMethod: Arbitrary[Method] = Arbitrary(
```

Comment:
We already have one of these in ArbitraryInstances in the testing project.

Reply:
👍 https://github.com/http4s/http4s/pull/3167/commits/59feb8bdb5585b3a0ed5bd503b79a61c67a2dcf5